Update upgrade-basic-load-balancer-on-aks.md (#277)
MicrosoftDocs/azure-aks-docs: articles/aks/upgrade-basic-load-balancer-on-aks.md

### Before You Begin

>[!Important]

> >Downtime occurs during migration. We recommend the user test the migration in dev/test environment before trying it out with a production cluster.

Reviewer: ```suggestion
>Downtime occurs during migration. We recommend you test the migration in a development or test environment before trying it out with a production cluster.
```
